.github/workflows/lint-tests-release.yml

```diff
@@ -63,10 +63,26 @@ jobs:
         working-directory: ./contract-examples
       - name: Run e2e tests
         shell: bash
-        run: E2E=true scripts/run.sh
+        run: MODE=test scripts/run.sh
+
+  simulator_test:
+    name: Load testing with simulator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
Can we rebase this off of master to include the go 1.18 version bump and bump this to go 1.18 as well?